h/templates/activity/search.html.jinja2

```diff
@@ -35,10 +35,20 @@
 <div class="search-bucket-stats">
   {% if bucket.uri %}
     <div class="search-bucket-stats__key">
-        Address
+        URL
     </div>
-    <div class="search-bucket-stats__val search-bucket-stats__address">
-        {{ bucket.uri }}
+    <div class="search-bucket-stats__val search-bucket-stats__url">
+        <div><a class="search-bucket-stats__link"
+                href="{{ bucket.uri }}">{{ pretty_link(bucket.uri) }}</a></div>
```

Comment:
Are there cases where a document bucket either doesn't have a URI or doesn't have one that we can / should link to? Locally-annotated files for example. I know that `DocumentHTMLPresenter.web_uri` always tries to return an http(s) URL for a document, and I think it'll return `None` if the document doesn't have an http(s) URL and will also strip `https://via.hypothes.is/` from the start